src/systems/ec/implementations/react/components/table/examples/TableSort.jsx

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/no-extraneous-dependencies, react/no-array-index-key  */
```

Comment:
The `react/no-array-index-key` ignore rule is available in in 26 occurrences in 24 files which is a hint the rules does not help much in the code base. Could you please remove these ignore rules and add a global one after this one please? https://github.com/ec-europa/europa-component-library/blob/v2-dev/.eslintrc.js#L17

Reply:
I wrongly suggested updating the root file. There are eslint config files for the react implementations which I updated to solve these boring issues